slo-reporter/overlays/cnv-prod/secrets.enc.yaml

```diff
@@ -3,7 +3,7 @@ items:
 -   apiVersion: v1
     data:
         token: ENC[AES256_GCM,data:rjG0MX4DXvInhIVoeASIMAPC3q2F5gCrii4nWfFpjK6VMLulZDUPaS7G+wQR3OUo21wnYByyAkh1/XPvyT2+coEY6pougjRMeRzcgguS8h1Vbgn/i+qjBbJZ41IcWlF/dQ+lVqLoPLPGMfnYiE3NGiDYBx4QS1+rWBtJejQibPqCVX4zmKz8wGq2HpoEvM7+pH9jdS5TOcVrlZsnPO/5QLFfuxsPQ9xY+Va12P6qZluQfnm84pOgqA8pb8qpdYMbZ2N/LgsHW6ToQAjB1EQ+d4H610BtPpu2lnx3mpnTQTGz92bio3xlRwVydXdCS3Tcx3SumeX1VkON0i81l6F1HHzYpqBB4mIBIW40EyanTs5r/Q9+pbyQAjl89yY0OVpwGaODma0h9KmS0dqPJIf4MBmedaUDRqaW2dm6/9fXQbNP6qw1olDYCMtq2MZCI2MmuLKeScK3rlOVsLa9jhoi16fA44UapbgnATVec0NRcZvzIrQ7bxF3nHxDchBKOnjo3FjlNDv7sr1Mr1B2wnImGiVyBgpqQtIwqcuRInhIAzqCN+JfNHl7vV9m4SfCTGhhMk4NV4BqFbN/t/ADNc2JEAb9uLoYN5bp4dQ+QkeN691P6SJjUo+ZudhY8W7nJZiO7AmILFtmhEK8LsE1RmYQEUM/wckXmZBK3UWRUzGTtur5M0C26zG2fZD5iigay5hNX+veIP4XyDoI/cISvD/zMWfSybx9MRuEG4kCk1Lk+MfTQXRWYrHtnHJLCDX0JIzsCw5s3MUZnGiubrKrLTLwBaDmO3GeTeXEpmvK1qvCcABo2qN5pmDxRTjxXXaQwYU+h/um3APZgJZqluJU6xwN68NQcI/LHAtzZJdgIhnnaNfaRFXPuBi3b1hZToca8SyuTsoP0FeRfZEz9GviBXHHfeurNKDu82c+/SmcrE2p4titYST5cHdLrX55ZXX/hSt1WY075pIzeRpJ2jEpkse7tRyM+3Nboioilte+X4EwzUa1f7+2IDq3Mwq+bjqnWhO2WNn/5vb0+a4VAgMNBEn/MM0R/bjcT2QFeFnSJuSZKH6Ou/3zsPShXurMRtAPE6rdx98NPahYNv0NlFuBMVtUtzVUCGRf4wzdxE2I8wZ4KBDSveWWndslDQtLKCA12ysTjPeSs9vqckgYtiFpv06Sp4F2W+vokfsxHuZh+w4x/sDHFvmuog/4Ajiq4H7L7fL/6IhdYYWT64jz6oVfmxvE8LqaES//IvyJa3qCZ2Z2h6kJdo0VJCEcLRYdQ1m7/md4veNH+ie+z6gn8Kl/0lxZF6YtCSn9QQD8r6M/F7EukW3QAsfsLKC9XM8KIUCyxGp73WaBTfWiva9L+OUl2jlWgpRGLEZjpw5+YW23bp/56Tj4yqxdNBAIY4bwAM3CFGrJviX6qsHFYeHnTY0IWyCBUVVjjZ3TsQdAcwXZiDtG3jpRxoe91F5cKi/WwyG424nI0IIhwA4kQ689Z3b+p9850z6FtXwPfGMufiNck6CxvjQrW6OY6HG4yroVz96xfBvwFGUVSRVpd/78Ao8T9wVulTGKYqGTIxSiUDIEzqhYo82H3VJOP1do6lksrC3Biy3D/Tug/0OiR/kegkEBO5tKOZv8GiPZlX2taSFayo2l7KE=,iv:cR2qa7USMltmXDfKQiqGA6yNOPbtlRL31fVF+ioXnkE=,tag:uhaH2peeGv1F7VlUU/D7FA==,type:str]
```

Comment:
is this token required ? wanted to know as the content of this might of the internal thanos.

Reply:
It is not required in prod, but we should keep the empty variable in configmap or SLO-reporter will fail

Reply:
ack, so what will happen if the same token stay will it break it ? or as you suggested we should change it with empty value, if so can please make that change in this pr.

Reply:
I changed it to empty :) Thanks @harshad16 !